modules/oe_whitelabel_multilingual/config/optional/block.block.openeuropa_content_language_switcher.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - oe_multilingual
+  theme:
+    - oe_whitelabel
+id: openeuropa_content_language_switcher
```

Comment:
lets name the file 
block.block.oe_whitelabel_language_switcher.yml
and the id
oe_whitelabel_language_switcher